Description: Temporarily disable +logging to make build
 No rdeps depend on +logging AFAICT
Last-Update: 2023-10-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -90,8 +90,8 @@
-[dev-dependencies.env_logger]
-version = "0.9.3"
-features = [
-    "atty",
-    "humantime",
-    "termcolor",
-]
-default-features = false
+#[dev-dependencies.env_logger]
+#version = "0.9.3"
+#features = [
+#    "atty",
+#    "humantime",
+#    "termcolor",
+#]
+#default-features = false
@@ -117,3 +117,3 @@
-    "aho-corasick?/logging",
-    "memchr?/logging",
-    "regex-automata/logging",
+#    "aho-corasick?/logging",
+#    "memchr?/logging",
+#    "regex-automata/logging",
--- a/tests/lib.rs
+++ b/tests/lib.rs
@@ -20 +20 @@
-    let _ = env_logger::try_init();
+    //let _ = env_logger::try_init();
